toolchest/yaml.py

```diff
@@ -4,6 +4,12 @@
 import yaml
 
 
+class IndentDumper(yaml.Dumper):
+
+    def increase_indent(self, flow=False, indentless=False):
+        return super().increase_indent(flow, False)
```

Comment:
Should the second argument be "indentless" instead of always overriding to False? Since that's the default anyway

Reply:
Not really, but hopefully my change will help clarify things.